bugsnag-android-core/src/main/java/com/bugsnag/android/Client.java

```diff
@@ -623,6 +651,12 @@ void notifyUnhandledException(@NonNull Throwable exc, Metadata metadata,
         Metadata data = Metadata.Companion.merge(metadataState.getMetadata(), metadata);
         Event event = new Event(exc, immutableConfig, handledState, data, logger);
         populateAndNotifyAndroidEvent(event, null);
+
+        // persist LastRunInfo so that on relaunch users can check the app crashed
+        int consecutiveLaunchCrashes = lastRunInfo == null ? 0
+                : lastRunInfo.getConsecutiveLaunchCrashes();
+        new LastRunInfo(consecutiveLaunchCrashes, true, false);
+        lastRunInfoStore.persist(lastRunInfo);
```

Comment:
Unless I'm missing something here, you're not updating `lastRunInfo` so it's just being re-persisted with the old values.

Reply:
Updated implementation so that it now persists the correct values.